OXRS-SHA-StateMonitor-ESP32-FW.ino

```diff
@@ -53,7 +56,10 @@
 
 /*--------------------------- Global Variables ---------------------------*/
 // Each bit corresponds to an MCP found on the IC2 bus
-uint8_t g_mcps_found = 0;
+uint8_t   g_mcps_found = 0;
+
+// temperature update interval timer
+uint32_t  g_last_temp_update = -TEMP_UPDATE_INTERVAL;

```

Comment:
Yes, this ensures that we get an update when the loop is the first time entered because `millisec() - g_last_temp_update` will be > `TEMP_UPDATE_INTERVAL`